test/sqllogictest/mzcompose.py

```diff
@@ -13,6 +13,10 @@
 SERVICES = [Postgres(), SqlLogicTest()]
 
 
+def workflow_default(c) -> None:
+    c.workflow("sqllogictest-fast")
+
+
```

Comment:
why forward this one? just to keep the names consistent down below?

Reply:
Yeah it just felt more clear here since there are multiple equivalently-important workflows. All the other ones where I renamed them the name of the workflow added no value compared to the name of the directory.

I might add doc parsing to mzcompose so that `mzcompose list` can provide explanations, which would eliminate this.

Reply:
It already does that! Add a doc comment and it will show up in `mzcompose list`.

Reply:
This is better work for a followup, but I'd say we should rename `slt.sh` to `slt-slow.sh`, `slt-fast.sh` to `slt.sh`, and then have one workflow called "default" that takes a `--slow` option.